project/assets/main/locale/es.po

```diff
@@ -12636,7 +12636,7 @@ msgstr "Cara"
 
 #: project/assets/main/locale/localizables-extracted.py:2653
 msgid "Eyes"
-msgstr ""
+msgstr "Ojos"
```

Comment:
Can we flag these translations as "requiring human review"?

Reply:
I've added the `#, fuzzy` flag which appears to be the standard way of marking something as "a human needs to take action"